docs/tutorial/application-distribution.md

```diff
@@ -55,6 +55,8 @@ electron/resources/
 
 More details can be found in [Application packaging](application-packaging.md).
 
+### Reducing app.asar file size
+By default when packing with asar it includes all files inluding the `electron.exe` used for testing which is quite large. To ignore these when packaging use ``--unpack-dir "**/dist"``.
```

Comment:
I'm not sure I can see where this would ever be the case.  If you are using `electron-packager` or even performing packaging manually you should be running `npm prune --production` on your modules before packaging (`electron-packager` does this for you).  This would remove any dev dependencies including the electron prebuilts.

The ignore pattern you've specified would also have untold side affects in other npm modules that also use a `dist` folder.